### ISSUE TITLE
xmlrpc library is incomatible with php 7

### ISSUE THREAD

**Issue by h3po** [bitnami/bitnami-docker-testlink#82]
The image was updated to php 7 but testlink 1.9.18 comes with an old version of phpxmlrpc which is incompatible with php 7.

steps to reproduce:
I'm not sure. After migrating to 1.9.18 some of our testplans work fine, some trigger this error:
```
==> /opt/bitnami/apache/logs/error_log <==                                                
[Wed Dec 12 15:11:15.900856 2018] [php7:emerg] [pid 161] [client 172.19.0.1:53082] PHP Parse error:  syntax error, unexpected 'new' (T_NEW) in /opt/bitnami/testlink/third_party/phpxmlrpc/lib/xmlrpc.inc on line 13
43, referer: http://localhost/lib/execute/execNavigator.php?setting_testplan=
```
how to fix:
replace `/opt/bitnami/testlink/third_party/phpxmlrpc` with a newer version or search-replace all `=\s*&\s*new` with `= new` in `/opt/bitnami/testlink/third_party/phpxmlrpc/lib/*.inc`.
I'd be surprised if there weren't more libraries that need updating to work with php7. However I'm working with testlink for the first time and also not an expert in php webapps.
Am I using the wrong version of the image? 
`
$ docker image ls | grep testlink
bitnami/testlink                                     1.9.18              0b390815349e        27 hours ago        371MB
`

**Comment by andresbono** [comment]
Hi @h3po, could you tell us from which version did you upgrade to TestLink 1.8.18?

**Comment by h3po** [comment]
Hi, Happy New Year.
I did not previously use the bitnami docker image for testlink. I started with a database dump of an old testlink installation on centos which was created many years ago... I just updated it with the migration scripts included in the docker image from 1.9.14 to 1.9.15 to 1.9.16 and so on. 
However I don't understand how the database is relevant. The image is built with php7 which some of the libraries shipped by testlink are not compatible with. Do you think the content of my db is somehow special in triggering the xmlrpc related errors?
